### ISSUE TITLE
Support SVN:global-ignores feature

### ISSUE THREAD

**Issue by netsrotr** [subclipse/subclipse#127]
Subversion has support for svn:global-ignores for a long time now (since v1.8) and described here for Apache Subversion: [Inheritable-Ignores svn:global-ignores](https://cwiki.apache.org/confluence/display/SVN/Inheritable-Ignores-AutoProps#Inheritable-Ignores-AutoProps-BehavioralSpecification). 
Also the the common tortoiseSVN client has yet support for that.

That would be of really help for our **big** projects!

**Comment by markphip** [comment]
We get support for features like this for "free". We do not implement these things ourselves it is done by the SVN API.  I am only guessing, but perhaps you are using the SVNKit provider and it does not support this feature? If so, then try JavaHL and see if that makes a difference.

**Comment by netsrotr** [comment]
Mhm, sound good. But: I already use JavaHL 1.11.1 (with Eclipse 2020.3), no effect there -sorry.

**Comment by markphip** [comment]
It works the way I expected it too.

I set the svn:global-ignores property at the root of my repository with some file extensions in it.  In a Subclipse working copy of /trunk I did an update.  Then created files with those extensions and they were ignored.

I did notice if I create a file and then later update the property to include that extension, the decorator in Subclipse for the file only refreshed after I touched it again, but that sort of behavior exists for a lot of scenarios.

Best I could tell, both JavaHL and SVNKit both worked the same.

**Comment by netsrotr** [comment]
Sorry, I was not aware of that:

> I did notice if I create a file and then later update the property to include that extension, the decorator in Subclipse for the file only refreshed after I touched it again, but that sort of behavior exists for a lot of scenarios.

Thanks for the fast help!

**Comment by markphip** [comment]
It is primarily the decorators that are impacted. There is nothing to trigger them to refresh until you touch the file.  As long as you have done Team > Update though I found that if I do any action like Team > Commit it will ignore the files even though the decorator is showing the ?

In my limited tests if I closed the project in Eclipse and then opened it again (after updating) then it did seem to refresh the decorators and show them as ignored.
